docs/source/supported/frontier.rst

```diff
@@ -96,21 +91,29 @@ not provided by the system)
    python3 -m venv ve.rp
    source ve.rp/bin/activate
 
-Install RADICAL-Pilot after activating a corresponding virtual environment.
+Install RADICAL-Pilot after activating a corresponding virtual environment:
+
+.. code-block:: bash
+
+   pip install radical.pilot
```

Comment:
@mturilli I actually thought to avoid it here, since we have it in Getting Started tutorial, and it is not a platform specific. That's why I thought just to refer install RCT should be enough